docs/NativeModulesIOS.md

```diff
@@ -4,7 +4,7 @@ title: Native Modules (iOS)
 layout: docs
 category: Guides
 permalink: docs/nativemodulesios.html
-next: testing
+next: debugging
 ---
```

Comment:
I ordered Debugging before Testing as one was user facing, and one was for contributors. I figure, you will generally use react before contributing.